src/main/scala/outwatch/dom/VDomModifier.scala

```diff
@@ -9,13 +9,31 @@ import snabbdom.{DataObject, VNodeProxy, hFunction}
 import scala.scalajs.js
 import collection.breakOut
 
-sealed trait VDomModifier_ extends Any
+sealed trait Modifier extends Any
```

Comment:
Lets keep this PR focused and not rename `VDomModifier`. Otherwise, we have more conflicts than needed on other branches.

Reply:
It seemed a good time to rename it because `VNode_` got removed and together with `VDomModifier_` they had names I didn't like. But it makes sense regarding conflict with other branches.